react-app/src/utils/leaguehelpers.js

```diff
@@ -74,10 +81,48 @@ export default {
 
 	convertMatchTime: (time) => {
 		//This is just for the schedules bc theyre always at night
+		//we can change it later..
 		let newTime = (parseInt(time) - 1200).toString();
 		return newTime.substr(0,1) +':' + newTime.substr(1,2) + 'pm';
 	},
 
-	
+	getMatchTimes: (league, dateId) => {
+
+		if(league === null || dateId === ''){
+			return null;
+		}
+
+		let matchTimes = [];
+
+		league.scheduledMatches.forEach((match) =>{
+			if(match.dateId === dateId){
+				matchTimes.push(match.matchTime);
+			}
+		});
+
+		return matchTimes;
```

Comment:
Before you return this you should sort it in asc order. It's a pretty good rule of thumb to not trust the order of the objects coming back in an API